### ISSUE TITLE
The results on the SRS1 dataset

### ISSUE THREAD

**Issue by Sample-design-alt** [Xiaoyu-Tao/HiTime#6]
Another question about SRS1 dataset. Because loss the crosstimenet checkpoint, so i replace the crosstimenet with Moment model.

1、
The stage 0 use **convtimenet**, the accuracy is 0.7747440273.
The stage 1&2 use **convtimenet** and **Moment**, the code is presented as below: 
`ts_encoder_ft = create_ConvTimeNet(config, precision)`
`ts_encoder_pt = Moment_model()`
Finally, the accuracy is 0.8361774744027304

2、Next
The stage 0 use **moment**, the accuracy is 0.8805460750853242.
The stage 1&2 use **Moment** and **Moment**, the code is presented as below: 
`
                ts_encoder_ft = Moment_model()
                ts_encoder_pt = Moment_model()`

Finally, the accuracy is 0.7337883959044369, 0.5017064846416383, 0.856655290102389

I have some question about the results. 
1. Why does using a larger foundational model like Moment actually lead to poorer performance?
2. Does using an LLM make it more likely for results to exhibit greater variance?

**Comment by pty12345** [comment]
Thanks for your interest in our work!

Could you please share the details of the MOMENT model you used (e.g., d_model, d_ff)?

Meanwhile, could you kindly clarify how you obtained the MOMENT parameters—were they trained on our specific datasets, or from another source?

**Comment by Sample-design-alt** [comment]
Thanks for your reply!
 Acctually,  I use the MOMENT_SMALL model to test the dataset. The parameters are obtained from the origin github repo https://github.com/moment-timeseries-foundation-model/moment/blob/main/momentfm/models/moment.py. 

The parameters is set as below:
`
"model": "moment",
"transformer_type": "encoder_only",
"d_model": null,
"task": "classification",
"seq_len": 512,
"max_seq_len": 512,
"fixed_max_len": true,
"gpt_layers": 6,
"patch_size": 8,
"stride": 8,
"patch_stride_len": 8,
"device": "cpu",
"transformer_backbone": "google/flan-t5-large",
"model_kwargs": {},
"freeze_encoder": false,
"freeze_embedder": false,
"freeze_head": false,
"enable_gradient_checkpointing": false,
"reduction": "concat",
"scalar_scales": null,
"epsilon_scalar_enc": 1.1,
"hidden_dim_scalar_enc": 16,
"t5_config": {                  ### here we do not change any parameter for t5 config
    "architectures": [
      "T5ForConditionalGeneration"
    ],
    "d_ff": 1024,
    "d_kv": 64,
    "d_model": 512,
    "decoder_start_token_id": 0,
    "dropout_rate": 0.1,
    "eos_token_id": 1,
    "feed_forward_proj": "gated-gelu",
    "initializer_factor": 1.0,
    "is_encoder_decoder": true,
    "layer_norm_epsilon": 1e-06,
    "model_type": "t5",
    "n_positions": 512,
    "num_decoder_layers": 8,
    "num_heads": 6,
    "num_layers": 8,
    "output_past": true,
    "pad_token_id": 0,
    "relative_attention_max_distance": 128,
    "relative_attention_num_buckets": 32,
    "tie_word_embeddings": false,
    "use_cache": true,
    "vocab_size": 32128
  }

`

**Comment by pty12345** [comment]
Hi, Sample-design-alt,

We consider that the Data-specific Encoder (i.e., ts_encoder_ft) may require a fine-grained classification model specifically trained on the target dataset. Using MOMENT-SMALL that has not been fine-tuned on the target dataset may lead to difficulties in the Qformer alignment stage, primarily due to mismatched classification features.

Therefore, we suggest training the MOMENT-SMALL on the classification dataset and testing it as the ts_encoder_ft.

**Comment by Cistarry** [comment]
@Sample-design-alt Hello, may I ask if the reproduction was successful?

**Comment by Sample-design-alt** [comment]
Thank you to the author for providing the solution.
In fact, I still find it difficult to reproduce the experimental results. Taking SRS1 as an example, after modifying it according to the author's suggestions, the experimental results were even worse than the original version. **Moment** stage0 accuracy: 0.825938566552901, stage 2 accuracy: 0.5221843003412969. Due to other commitments recently, this matter has been temporarily put on hold. I hope my findings prove helpful to you.
